### DISCUSSION TITLE
Can i install chocolatey from the source code on my comuter？

### DISCUSSION THREAD

**Discussion by buzhi-kefou** [chocolatey/choco#3210] [Q&A]
I have ran the build.bat successfully， but i don't know how to install Choco with the source code.

**Comment by gep13** [comment]
@buzhi-kefou can I ask, what is the goal of building directly from source code?  Why not use one of the official releases of Chocolatey CLI?

If you continue to want to install from source, you will need to follow some of the step in the `More Install Options` section here: https://docs.chocolatey.org/en-us/choco/setup#more-install-options but I don't think we have exactly steps for installing from source code built versions.

**Reply by buzhi-kefou** [reply]
My company want to build internal package manager. So i want to  compile chocolatey and write our business.

**Reply by gep13** [reply]
I am not sure I follow exactly what you are trying to achieve, but it does sound like this would be an unsupportable usage of Chocolatey, and therefore we won't be able to provide much guidance, other than what is already available on the docs site that I linked to.
